pkg/db/src/implementations/version-1.2/books.ts

```diff
@@ -37,10 +38,21 @@ class Books implements BooksInterface {
 	}
 
 	async get(isbns: string[]): Promise<(BookEntry | undefined)[]> {
-		const rawBooks = await this.#db._pouch.allDocs<BookEntry>({ keys: isbns.map((isbn) => `books/${isbn}`), include_docs: true });
-		// The rows are returned in the same order as the supplied keys array.
-		// The row for a nonexistent document will just contain an "error" property with the value "not_found".
-		return unwrapDocs(rawBooks);
+		const books = await this.#db._pouch
+			.allDocs<BookEntry>({ keys: isbns.map((isbn) => `books/${isbn}`), include_docs: true })
+			// The rows are returned in the same order as the supplied keys array.
+			// The row for a nonexistent document will just contain an "error" property with the value "not_found".
+			.then((docs) => unwrapDocs(docs));
+
+		const isbnsToFetch = wrapIter(isbns)
+			.zip(books)
+			.filter(([, book]) => book === undefined)
+			.map(([isbn]) => isbn)
+			.map((isbn) => pluginManager.getBook(isbn));
```

Comment:
Let's move this last line to Promise.all (after the array has been materialised) - not that this is wrong, but it would be cleaner